src/main/java/org/spongepowered/api/data/DataRegistration.java

```diff
@@ -118,6 +120,21 @@ static Builder builder() {
      */
     PluginContainer getPluginContainer();
 
+    /**
+     * Creates a DataRegistration for a single key with a DataStore for given data-holders.
+     *
+     *
+     * @param key the data key
+     * @param dataHolders the data-holders
+     * @param <T> the keys value type
+     *
+     * @return The built data registration
+     */
+    static <T> DataRegistration of(Key<Value<T>> key, Class<? extends DataHolder>... dataHolders) {
```

Comment:
Are you intending to be able to call `DataRegistration.of(Keys.FOO)`? because the varargs can have a length of `0` by omitting the argument altogether in this case (this is why Cause historically always had an `Object, Object...` method signature, to ensure there's at least one argument passed in.